build/changeSets/changeSignatureUitype.php

```diff
@@ -0,0 +1,24 @@
+<?php
+class changeSignatureUitype extends cbupdaterWorker {
+	function applyChange() {
+		if ($this->isBlocked()) return true;
+		if ($this->hasError()) $this->sendError();
+		if ($this->isApplied()) {
+			$this->sendMsg('Changeset '.get_class($this).' already applied!');
+		} else {
+			$moduleInstance = Vtiger_Module::getInstance('Users');
+			$block = Vtiger_Block::getInstance('LBL_SIGNATURE', $moduleInstance);
+			if (!$block) {
+				$block = new Vtiger_Block();
+				$block->label = 'LBL_SIGNATURE';
+				$block->sequence = 2;
+				$moduleInstance->addBlock($block);
+            }
+            $block = Vtiger_Block::getInstance('LBL_SIGNATURE', $moduleInstance);
```

Comment:
Incorrect formatting. Please remember to apply [coreBOS code formatting rules](http://corebos.org/documentation/doku.php?id=en:devel:developmentguidelines#code_formatting_and_validation_tools)